Sources/String.swift

```diff
@@ -81,7 +81,7 @@ internal extension String {
     }
     
     subscript(range: Range<Int>) -> String {
-        return self[characters.index(startIndex, offsetBy: range.lowerBound)..<characters.index(startIndex, offsetBy: range.upperBound)]
+      return String(self[characters.index(startIndex, offsetBy: range.lowerBound)..<characters.index(startIndex, offsetBy: range.upperBound)])
```

Comment:
I guess with new String API it would make sense to revise this extension and get rid of subscripts completely. What do you think?